### DISCUSSION TITLE
Feature Requests (info)

### DISCUSSION THREAD

**Discussion by jsmorabito** [athensresearch/athens#826] [General]
🚧 UNDER CONSTRUCTION 🚧

1. Search [Github Issues](https://github.com/athensresearch/athens/issues) and [Github Discussions](https://github.com/athensresearch/athens/discussions/categories/feature-requests) to see if your feature has already been requested 

2. If you don't see your feature requested in either of those locations make a new post in the [feature requests discussion section](https://github.com/athensresearch/athens/discussions/categories/feature-requests)

3. Upvote, comment, provide details for feature requests in the [feature request discussion section](https://github.com/athensresearch/athens/discussions/categories/feature-requests)

Requests will be turned into issues on three occasions:

1. Someone is going to begin working on the feature

2. The core team puts the feature on the [roadmap](https://github.com/athensresearch/athens/projects/12)

3. A Gitcoin bounty is made for the feature (gitcoin bounties guide (TBD))

After the feature request has been completed, the github discussion will be closed (not possible yet, please bother the github team here to get this functionality https://github.com/github/feedback/discussions/3097) 

*this is a draft for a new feature request process please comment here if you have suggestions for improving it!

**Comment by Ejipuh** [comment]
Do we have a resource that gives guidelines on what makes a good or helpful feature request?

**Comment by m-levy** [comment]
@tangjeff0 Is there a way to create metadata templates and attributes similarly to: https://padminipyapali.medium.com/using-templates-in-roam-research-ee6bd480f3e0

**Reply by ghost** [reply]
this is not yet available. feel free to add to our ongoing discussion about templates here: https://github.com/athensresearch/athens/discussions/875
